packages/blade/src/utils/makeBorderSize/makeBorderSize.test.ts

```diff
@@ -0,0 +1,17 @@
+import border from '../../tokens/global/border';
+import makeBorderSize from '.';
+
+describe('makeBorderSize', () => {
+  it('should return the border width in `px`', () => {
+    const space = makeBorderSize(border.width.thin);
+    expect(space).toEqual('1px');
+  });
+  it('should return the border radius in `px`', () => {
+    const space = makeBorderSize(border.radius.small);
+    expect(space).toEqual('2px');
+  });
```

Comment:
also i think we shall restrict this use case and only stick with `%` or number

Reply:
What do you mean? `border.radius.small` is a `number` here. So we are already doing % or number